src/components/Quote.vue

```diff
@@ -127,6 +128,12 @@ export default {
 	},
 
 	computed: {
+		component() {
+			return this.canCancel || this.editMessage
+				? { tag: 'div', link: undefined }
```

Comment:
Why use div on edit? Seems useful to navigate to the message being edited.

Reply:
You're right, edit mode doesn't have interactive elements inside